### PR TITLE
Update tooling

### PR DESCRIPTION
**Pull request by Klynger**
#### How to test it?

[Workspace](https://summarytooling--storecomponents.myvtex.com/)
[ProductSummarySKUSelector example](https://productsummarylint--worldwidegolf.myvtex.com/stick?_q=stick&map=ft)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+  "root": true,
   "env": {
-    "browser": true,
+    "node": true,
     "es6": true,
     "jest": true
-  }
-}
+  },
+  "parser": "babel-eslint"
```
Comment (Klynger) [inline .eslintrc:9]: This is needed to support class components

File: react/components/ProductSummary.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React, { useCallback, useMemo, useEffect } from 'react'
+import path from 'ramda/src/path'
```
Comment (Klynger) [inline react/components/ProductSummary.js:2]: https://github.com/ramda/ramda#usage

File: react/components/ProductSummaryImage/ProductImage.js [hunk 3]
```diff
@@ -18,17 +20,19 @@ const CSS_HANDLES = ['image', 'imageContainer', 'product', 'imagePlaceholder']
...
+
   if (aspectRatio) {
     return imageUrl(src, DEFAULT_SIZE, MAX_SIZE, aspectRatio)
   }
+
   return src
 }
 
-const getStyle = (width, height, aspectRatio, maxHeight) => {
+const getStyle = ({ width, height, aspectRatio, maxHeight }) => {
```
Comment (Klynger) [inline react/components/ProductSummaryImage/ProductImage.js:35]: These functions are used only in one place

File: react/utils/aspectRatioUtil.js [hunk 4]
```diff
@@ -4,24 +4,27 @@ import { changeImageUrlSize } from './normalize'
...
+
     if (data.length !== 2) {
       return null
     }
 
     const [width, height] = data
     const ratio = parseFloat(height) / parseFloat(width)
 
-    if (typeof ratio !== 'number' || isNaN(ratio)) {
+    if (typeof ratio !== 'number' || Number.isNaN(ratio)) {
```
Comment (kaisermann) [inline react/utils/aspectRatioUtil.js:27]: `Number.isNaN` will literally check if a value is `NaN` and if it's not a number

**Pull request merged**